utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -607,6 +607,7 @@ impl State {
 	pub(crate) async fn into_ext<Block: BlockT + DeserializeOwned, HostFns: HostFunctions>(
 		&self,
 		shared: &SharedParams,
+		command: &Command,
```

Comment:
I don't get why you make it so complicated? You can just pass some `ensure_try_runtime: bool` and everything passes `true`, but not `create_snapshot`.